src/util/contracthash.rs

```diff
@@ -12,11 +12,11 @@
 // If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
 //
 
-//! Pay-to-contract-hash support
+//! Pay-to-contract-hash support.
```

Comment:
This module is kind of deprecated, so maybe it is worth mentioning it in the docs

Reply:
Added a comment to the rustdocs of `contracthash` module. Done as a separate patch.